prodtests/full-system-test/aggregator-workflow.sh

```diff
@@ -44,67 +44,120 @@ if [[ $BEAMTYPE == "PbPb" ]]; then
 fi
 
 # Calibration workflows
-if ! workflow_has_parameter CALIB_LOCAL_INTEGRATED_AGGREGATOR; then WORKFLOW=; fi
+if ! workflow_has_parameter CALIB_LOCAL_INTEGRATED_AGGREGATOR; then
+    WORKFLOW=
+else
+    if [[ -z $AGGREGATOR_TASKS ]]; then
+  ech HERE1
+    AGGREGATOR_TASKS=ALL
+  fi
+fi
```

Comment:
Seems to be a typo `ech`, but anyhow this seems to be debug output that should go.
Second, I would always run the check for the aggregator task, also in case `CALIB_LOCAL_INTEGRATED_AGGREGATOR` is set

Reply:
Well, if it is not set, it will also default to `ALL`.

Reply:
Only if we have the aggregator running locally. Otherwise, mixing proxies will not work.

Reply:
I don't understand what you mean. This setting specified which input proxies are created, and which calib tasks are running.
So in my opinion, it makes sense with and without proxies.
Without proxies, it will simply run only a subset of the aggregation tasks. The default would be `ALL`, so by default it would run all tasks.

Reply:
I am then not understanding. I thought that we can run all the tasks only if we have no proxy, otherwise there will be too many input proxies in the aggregator.

Reply:
ok, I see that point.
But what will happen if AGGREGATOR_TASK is not set in that case? Will it just produce an empty workflow? I think that would be ok.
Or we add a check and throw an error message if `AGGREGATOR_TASK` is not set.

Still, the `export AGGREGATOR_TASKS=ALL` in `dpl-workflow.sh` is not needed, since it will default to `ALL`  in `aggregator-workflow.sh`

Reply:
I was thinking about adding an error if "AGGREGATOR_TASK" is not set. 
And you are right on the second point, it is not needed in dpl-workflow - I added it there before adding it in aggregator-workflow.